pkg/util/kubeconfig.go

```diff
@@ -312,3 +313,16 @@ func GetPortFromKubeConfig(filename, machineName string) (int, error) {
 	port, err := strconv.Atoi(kport)
 	return port, err
 }
+
+//UnsetCurrentContext unsets the current-context from minikube to "" on minikube stop
+func UnsetCurrentContext(filename, machineName string)  error {
```

Comment:
Please run `gofmt -s -w` on this file to reformat it to Go standards. Consider configuring your code editor to do this on save.

Reply:
formatted this one.